### DISCUSSION TITLE
PPSX file - Advice

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#258] [General]
Hi all,

The branch office wants us to play a PPTX (PowerPoint) file during our local needs in two weeks time. While I have Microsoft 365 at home, the Kingdom Hall computer doesn't, and I don't want to login there as it will get complicated.

I have searched on-line and apparently Apache "Open Office" plays PPSX files but I have been unable to get it to work.

The other suggestion is to use "PowerPoint Mobile", but I cannot get that to work very well either because I don't have a login account at the Kingdom Hall so it only plays on one screen, not the auditorium screens unless I drag the application onto those and I don't want to do that as it looks unprofessional when selecting the file etc. 

Microsoft encourages the use of via PowerPoint online / browser if you don't have the software, but again you need to login and the Kingdom Hall computer is kept very simple without too much in the way of account logins as per the PC setup instructions from the branch.

I thought VLC might be able to handle it, but again no.

I tried OnlyM (by Antony Corbett) but that also is unable to display the file. 

I will use the PowerPoint Mobile if necessary, but just wondered if there was a better, perhaps open source, method that will have display on the auditorium screens while controlled from the booth computer. 

Any suggestions / advice?

Kind regards
Simon

**Comment by sircharlo** [comment]
Have you tried LibreOffice?
If not, my other thought would be to convert the PPSX file to PDF; JWMMF can make quick work of a PDF file.

**Reply by Chipperchap** [reply]
Thank you for your reply.
I downloaded LibreOffice and it does play the file!
Because the file content is so dynamic, LibreOffice is unable to display some elements, but it is acceptable as the general audience viewers would not notice the missing slide transitions. 
It is because of the slide transitions I was reluctant to convert to PDF - so as to honour the authors efforts.
The alternative is to record the slideshow and convert to Mp4, and just keep the narration in sync with the slides, which is entirely feasible with some practice.

Interestingly JWMMF could not handle some pictures that were sent to me for a public talk in PDF format, but created by a Mac computer. I wonder what the difference is between regular PDF and Mac PDF? 
I ended up either finding the original images and using them, or in one case because I could not find the image I took a screenshot of the PDF and converted to PNG / JPG.

**Reply by sircharlo** [reply]
Glad you found something that worked! 

If ever you have that PDF file handy and don't mind sending it to me, I wouldn't mind taking a look at it.

**Reply by Chipperchap** [reply]
[01 Evening meal .pdf](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8504458/01.Evening.meal.pdf)
[02 Aged Adam and Eve .pdf](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8504460/02.Aged.Adam.and.Eve.pdf)
[03 Paradise .pdf](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8504461/03.Paradise.pdf)
[04 Welcoming back .pdf](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8504455/04.Welcoming.back.pdf)
[05 Emblems .pdf](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8504457/05.Emblems.pdf)

**Reply by sircharlo** [reply]
Odd. It converted without issue for me. I think this might be related to an underlying dependency update; in any case PDF files should always convert flawlessly. 

Thanks for sharing the files.

![image](https://user-images.githubusercontent.com/1275666/164049225-c79c9730-b989-48d8-9e22-02c75359324b.png)
